Tools/TestWebKitAPI/Tests/WTF/WorkQueue.cpp

```diff
@@ -285,16 +286,17 @@ TEST(WTF_WorkQueue, DestroyDispatchedOnDispatchQueue)
     public:
         DestructionWorkQueueTester(std::atomic<size_t>& counter, WorkQueue& queue)
             : m_counter(counter)
-            , m_ownerAssertion(queue.threadLikeAssertion()) // Queue is not yet current, but we expect it to be the time destructor runs.
+            , m_queue(queue) // Queue is not yet current, but we expect it to be the time destructor runs.
         {
         }
         ~DestructionWorkQueueTester()
         {
             m_counter++;
+            RELEASE_ASSERT(m_queue->isCurrent());
         }
     private:
         std::atomic<size_t>& m_counter;
-        NO_UNIQUE_ADDRESS ThreadLikeAssertion m_ownerAssertion;
+        Ref<WorkQueue> m_queue;
```

Comment:
I think this hunk you should remove, e.g. the functionality being used is the threadlikeassertion. So no need for the queue ref.

Reply:
the idea was for the test to work even with release build